src/click/decorators.py

```diff
@@ -126,10 +126,8 @@ def new_func(*args, **kwargs):  # type: ignore
 
 @t.overload
 def command(
-    name: t.Optional[str] = None,
-    cls: t.Type[CmdType] = ...,
-    **attrs: t.Any,
-) -> t.Callable[..., CmdType]:
+    __name: t.Callable[..., t.Any],
```

Comment:
```suggestion
    __func: t.Callable[..., t.Any],
```
To be consistent with changes in `core.py`.